src/GraphQL/Queries/EntryQuery.php

```diff
@@ -5,10 +5,15 @@
 use GraphQL\Type\Definition\Type;
 use Statamic\Facades;
 use Statamic\Facades\GraphQL;
+use Statamic\GraphQL\Queries\Concerns\FiltersQuery;
 use Statamic\GraphQL\Types\EntryInterface;
+use Statamic\GraphQL\Types\JsonArgument;
+use Statamic\Tags\Concerns\QueriesConditions;
 
 class EntryQuery extends Query
 {
+    use QueriesConditions, FiltersQuery;
```

Comment:
Just `use FiltersQuery`. Remove `QueriesConditions`.

We'll `use` that from the trait instead.